acceptance/bundle/telemetry/deploy-no-uuid/script

```diff
@@ -2,10 +2,7 @@ trace $CLI bundle deploy
 
 trace cat out.requests.txt | jq 'select(has("path") and .path == "/telemetry-ext") | .body.protoLogs.[] | fromjson'
 
-# Disable pipefail. head will skip reading all input once it encounters a newline. Not disabling pipefail will trigger
-# a SIGPIPE in linux based systems.
-set +o pipefail
-cmd_exec_id=$(cat out.requests.txt | jq '.headers."User-Agent".[0]'| head -n 1 | grep -o 'cmd-exec-id/[^ ]*' | cut -d '/' -f2)
+cmd_exec_id=$(extract_command_exec_id.py)
 
 update_file.py output.txt $cmd_exec_id  '[CMD-EXEC-ID]'
```

Comment:
How does this work? With output.txt is still being written to by test runner.

Reply:
It works because the python script waits for a complete JSON object to be serialized.